src/KDSoapClient/KDSoapClientInterface.cpp

```diff
@@ -284,6 +295,16 @@ void KDSoapClientInterfacePrivate::setupReply(QNetworkReply *reply)
     }
 }
 
+bool KDSoapClientInterfacePrivate::useWsAddressing() const
+{
+    return m_useWsAddressing;
+}
+
+void KDSoapClientInterfacePrivate::setUseWsAddressing(bool useWsAddressing)
+{
+    m_useWsAddressing = useWsAddressing;
+}
+
```

Comment:
I think these duplicate setters are not nessesary, this could be done directly in KDSoapClientInterface::useWsAddressing

Reply:
Good point! Fixed.